### DISCUSSION TITLE
Avalonia.Browser 11.0.11 UP  11.1.0-beta2  ERROR

### DISCUSSION THREAD

**Discussion by YSW33** [AvaloniaUI/Avalonia#16327] [General]
### Is your feature request related to a problem? Please describe.

Avalonia.Browser 11.0.11 UP  11.1.0-beta2
Cannot find namespace  Avalonia.Browser

![image](https://github.com/user-attachments/assets/e4e890c9-c5f8-4e2a-9850-dd6aa6347624)

### Describe the solution you'd like

Avalonia.Browser 11.0.11 UP  11.1.0-beta2

**Comment by rabbitism** [comment]
What are your package references?

**Comment by YSW33** [comment]
> 您的软件包参考是什么？

Project usage Avalonia.Templates  VS2022。
Avalonia.Templates  is Avalonia 11.0.6

**Comment by maxkatz6** [comment]
Change TargetFramework from "net8.0" to "net8.0-browser". Or recreate this specific project from the latest template (with 11.1.0-beta1 flag, just change version to rc later)
